packages/app-page-builder-elements/src/renderers/form/FormRender/functions/getNextStepIndex.ts

```diff
@@ -0,0 +1,218 @@
+import includes from "lodash/includes";
+import endsWith from "lodash/endsWith";
+import startsWith from "lodash/startsWith";
+import eq from "lodash/eq";
+import lte from "lodash/lte";
+import lt from "lodash/lt";
+import gte from "lodash/gte";
+import gt from "lodash/gt";
+
+import { FbFormRule, FbFormCondition } from "~/types";
+
+interface Props {
+    formData: Record<string, any>;
+    rule: FbFormRule;
+}
+
+const includesValidator = (filterValue: string, fieldValue: string) => {
+    if (fieldValue === null) {
+        return;
+    }
+
+    return includes(fieldValue, filterValue);
+};
+
+const startsWithValidator = (filterValue: string, fieldValue: string) => {
+    if (fieldValue === null) {
+        return;
+    }
+
+    return startsWith(fieldValue, filterValue);
+};
+
+const endsWithValidator = (filterValue: string, fieldValue: string) => {
+    if (fieldValue === null) {
+        return;
+    }
+
+    return endsWith(fieldValue, filterValue);
+};
+
+const isValidator = (filterValue: string, fieldValue: string | string[]) => {
+    if (fieldValue === null) {
+        return;
+    }
+
+    // This is check for checkboxes.
+    if (typeof fieldValue === "object") {
+        return fieldValue.includes(filterValue);
+    } else {
+        return eq(fieldValue, filterValue);
+    }
+};
+
+const gtValidator = ({
+    filterValue,
+    fieldValue,
+    equal = false
+}: {
+    filterValue: string;
+    fieldValue: string;
+    equal?: boolean;
+}) => {
+    if (fieldValue === null) {
+        return;
+    }
+
+    if (!equal) {
+        return gt(fieldValue, filterValue);
+    } else {
+        return gte(fieldValue, filterValue);
+    }
+};
+
+const ltValidator = ({
+    filterValue,
+    fieldValue,
+    equal = false
+}: {
+    filterValue: string;
+    fieldValue: string;
+    equal?: boolean;
+}) => {
+    if (fieldValue === null) {
+        return;
+    }
+
+    if (!equal) {
+        return lt(filterValue, filterValue);
+    } else {
+        return lte(fieldValue, filterValue);
+    }
+};
+
+const timeGtValidator = ({
```

Comment:
Can we use `@webiny/validation` lib here? I see a lot of validators here that the lib already has.

Reply:
I am working on it.